frontend/src/screens/LikedVideos/liked.module.css

```diff
@@ -0,0 +1,54 @@
+.liked__container{
+    margin-top: 1rem;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: column;
+}
+
+.liked__video{
+    position: relative;
+    margin-top: 1rem;
+    max-width: 100%;
+    max-height: 100%;
+    display: flex;
+    flex-direction: column;
+    gap: 0.5rem;
+    padding: 10px;
+    box-shadow: rgba(0, 0, 0, 0.1) 0px 10px 15px -3px, rgba(0, 0, 0, 0.05) 0px 4px 6px -2px;
+}
+
+.liked__video_delete{
+    display: flex;
+    flex-direction: row;
```

Comment:
flex-direction by default is row :)
This line can be removed

Reply:
done